enterprise/frontend/test/metabase-enterprise/sandboxes/sandboxes.cy.spec.js

```diff
@@ -26,27 +28,9 @@ const {
 } = SAMPLE_DATASET;
 
 const { ALL_USERS_GROUP, DATA_GROUP, COLLECTION_GROUP } = USER_GROUPS;
+const { sandboxed } = USERS;
 
-// TODO: If we ever have the need to use this user across multiple tests, extract it to `__support__/cypress`
-const sandboxed_user = {
-  first_name: "User",
-  last_name: "1",
-  email: "u1@metabase.test",
-  password: "12341234",
-  login_attributes: {
-    user_id: "1",
-    user_cat: "Widget",
-  },
-  // Because of the specific restrictions and the way testing dataset was set up,
-  // this user needs to also have access to "collections" (group_id: 4) in order to see saved questions
-  group_ids: [ALL_USERS_GROUP, COLLECTION_GROUP],
-};
-
-const [ATTR_UID, ATTR_CAT] = Object.keys(sandboxed_user.login_attributes);
-
-function createUser(user) {
-  return cy.request("POST", "/api/user", user);
-}
+const [ATTR_UID, ATTR_CAT] = Object.keys(sandboxed.login_attributes);
```

Comment:
I know this isn't new with this PR, but just as a comment: I might import `ATTR_UID` and `ATTR_CAT` directly. It's low risk in a test, but I'm always nervous when relying on the order of keys in an object. The key order usually doesn't matter, so other editors might insert new keys without thinking about it.

Reply:
How do you mean to "import them directly"? Literary export them first as `const`s and then import from that file?

Also, do you think this was even necessary in the first place? It might have been an overkill since we're always using `user_id` and `user_cat`? Thinking about it now, I might have named them `attr_uid` and `attr_cat` and then use them directly.

Reply:
Yup that's what I meant. Define them in one place and import wherever they're needed.

> do you think this was even necessary in the first place?

I think extracting reused strings into constants is a good practice but not necessary here. I think the MB codebase does too little of that overall.

Reply:
Ok, I removed this completely and went for using user attributes in the test directly.